lang/en.json

```
@@ -475,13 +478,29 @@
     "SelectOriginAltMode": "Origin Alt Modes",
     "SelectOriginEssence": "Origin Essences",
     "SelectOriginSkill": "Origin Skills",
+    "SelectShieldPrompt": "Select a Shield Option",
+    "SelectShieldTitle": "Shield Options",
     "SelectSkill": "Select a Skill.",
     "SelectUpgradeOrWeaponEffect": "Weapon Effect or Upgrade",
     "SelectWeaponAttach": "Select the Weapon.",
     "SelectionLimit": "Selection Limit",
+    "ShieldActiveBonus1": "Active Bonus 1",
+    "ShieldActiveBonus2": "Active Bonus 2",
+    "ShieldActiveEffectType": "Active Effect Type",
+    "ShieldCurrentStateActive": "Current mode: Active",
+    "ShieldCurrentStatePassive": "Current mode: Passive",
+    "ShieldDefenseBonus": "Applies 1 defense bonus",
+    "ShieldDefenseBonusCombo": "Applies 2 defense bonuses",
+    "ShieldDefenseBonusMixed": "Applies 1 defense bonus or 1 other type of bonus",
+    "ShieldDefenseBonusOption": "Applies 1 of 2 defense bonuses",
+    "ShieldNotEquipped": "This shield must be equipped to become activated.",
+    "ShieldOther": "Applies 1 other type of bonus",
```

```suggestion
    "ShieldDefense1Bonus": "Applies 1 defense bonus",
    "ShieldDefense1Of2Bonuses": "Applies 1 of 2 defense bonuses",
    "ShieldDefense1OtherBonus": "Applies 1 other type of bonus",
    "ShieldDefense2Bonuses": "Applies 2 defense bonuses",
    "ShieldDefenseMixedBonuses": "Applies 1 defense bonus or 1 other type of bonus",
    "ShieldNotEquipped": "This shield must be equipped to become activated.",
```
The diff looks bad but I just fixed the var names and sorted